versioned_docs/version-1.3.0/use-cases/distributed-mongodb-for-multicloud.mdx

```diff
@@ -0,0 +1,1069 @@
+
+   Example
+   ```
+   kubectl get pods -n mongodb-operator
+   ```
+
+   Example Output
+   ```
+   NAME                                           READY   STATUS    RESTARTS   AGE
+   mongodb-enterprise-operator-68cb5dd658-v2wrf   1/1     Running   0          51m
+   ops-manager-0                                  1/1     Running   0          8m36s
+   ops-manager-backup-daemon-0                    1/1     Running   0          2m15s
+   ops-manager-db-0                               3/3     Running   0          2m55s
+   ops-manager-db-1                               3/3     Running   0          3m49s
+   ops-manager-db-2                               3/3     Running   0          4m30s
+   ```
+5. Verify the created volumes for Ops Manager using the following command:
+
+   Example
+   ```
+   kubectl -n mongodb-operator get pvc
+   ```
+
+   Example Output
+   ```
+   NAME                               STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+   data-ops-manager-db-0              Bound    pvc-33b74868-f61f-499b-8c7d-ce61782aac88   15Gi       RWO            standard       23m
+   data-ops-manager-db-1              Bound    pvc-c0962ecc-b374-4e36-a17a-d7f404176c30   15Gi       RWO            standard       22m
+   data-ops-manager-db-2              Bound    pvc-b237acf9-6333-415c-9115-175f67ded4d6   15Gi       RWO            standard       21m
+   head-ops-manager-backup-daemon-0   Bound    pvc-2bd40fdf-2971-4250-b0d8-47315485b4bf   30Gi       RWO            standard       4m2s
+   ```
+6. (Optional) Verify the statefulsets created by the operator using the following command:
+
+   Example
+   ```
+   kubectl -n "mongodb-operator" get sts
+   ```
+
+   Example Output
+   ```
+   NAME                        READY   AGE
+   ops-manager                 1/1     12m
+   ops-manager-backup-daemon   1/1     6m
+   ops-manager-db              3/3     13m
+   ```
+
+7. (Optional) Verify the external service created for accessing the Ops Manager using the following command:
+
+   Example
+   ```
+   kubectl -n "mongodb-operator" get svc
+   ```
+
+   Example Output
+   ```
+   NAME                            TYPE           CLUSTER-IP    EXTERNAL-IP    PORT(S)                          AGE
+   operator-webhook                ClusterIP      10.7.47.251   <none>         443/TCP                          56m
+   ops-manager-backup-daemon-svc   ClusterIP      None          <none>         8443/TCP                         7m13s
+   ops-manager-db-svc              ClusterIP      None          <none>         27017/TCP                        14m
+   ops-manager-svc                 ClusterIP      None          <none>         8080/TCP                         13m
+   ops-manager-svc-ext             LoadBalancer   10.7.32.125   34.23.212.14   8080:31348/TCP,25999:31914/TCP   13m
+   ```
+
+8. To generate Ops Manager URL address if service is exposed as a LoadBalancer using the following command:
```

Comment:
If you read the sentence twice, you can figure out the punctuation error. 

There should be a comma after LoadBalancer and `using` should be `use`

Rephrase it as 

`To generate the Ops Manager URL address if the service is exposed as a LoadBalancer, use the following command:`